无法运行？

six-cli_V0.0.4-alpha_linux_amd64.tar.gz

同样的问题，直接运行二进制文件没有反应，没有日志
系统：unbuntu 16.04/18.04 
文件：ubuntu six-cli_V0.0.6-alpha_linux_amd64.tar.gz

直接编译报错
`# github.com/Mrs4s/power-liner
go/src/github.com/Mrs4s/power-liner/shell.go:144:32: undefined: strings.ReplaceAll
# github.com/Mrs4s/go-http-downloader
go/src/github.com/Mrs4s/go-http-downloader/downloader.go:35:20: unknown field 'MaxConnsPerHost' in struct literal of type http.Transport
# github.com/Mrs4s/go-six-cloud-upload-sdk
go/src/github.com/Mrs4s/go-six-cloud-upload-sdk/uploader.go:32:20: unknown field 'MaxConnsPerHost' in struct literal of type http.Transport`

go版本为`go1.10 linux/amd64`

> 同样的问题，直接运行二进制文件没有反应，没有日志
> 系统：unbuntu 16.04/18.04
> 文件：ubuntu six-cli_V0.0.6-alpha_linux_amd64.tar.gz
> 
> 直接编译报错
> 
> ```
> go/src/github.com/Mrs4s/power-liner/shell.go:144:32: undefined: strings.ReplaceAll
> # github.com/Mrs4s/go-http-downloader
> go/src/github.com/Mrs4s/go-http-downloader/downloader.go:35:20: unknown field 'MaxConnsPerHost' in struct literal of type http.Transport
> # github.com/Mrs4s/go-six-cloud-upload-sdk
> go/src/github.com/Mrs4s/go-six-cloud-upload-sdk/uploader.go:32:20: unknown field 'MaxConnsPerHost' in struct literal of type http.Transport```
> 
> go版本为`go1.10 linux/amd64`
> ```

运行无反应的问题还在排查
编译要求最低的go版本是1.13, 请升级一下

> > 同样的问题，直接运行二进制文件没有反应，没有日志
> > 系统：unbuntu 16.04/18.04
> > 文件：ubuntu six-cli_V0.0.6-alpha_linux_amd64.tar.gz
> > 直接编译报错
> > ```
> > go/src/github.com/Mrs4s/power-liner/shell.go:144:32: undefined: strings.ReplaceAll
> > # github.com/Mrs4s/go-http-downloader
> > go/src/github.com/Mrs4s/go-http-downloader/downloader.go:35:20: unknown field 'MaxConnsPerHost' in struct literal of type http.Transport
> > # github.com/Mrs4s/go-six-cloud-upload-sdk
> > go/src/github.com/Mrs4s/go-six-cloud-upload-sdk/uploader.go:32:20: unknown field 'MaxConnsPerHost' in struct literal of type http.Transport```
> > 
> > go版本为`go1.10 linux/amd64`
> > ```
> 
> 运行无反应的问题还在排查
> 编译要求最低的go版本是1.13, 请升级一下

更换版本后可以编译，生成的二进制文件可以正常使用，但是运行并退出后会影响ssh命令行中正常命令的显示，这个问题在unbuntu16.04和ubuntu 18.04上都出现了。

> > > 同样的问题，直接运行二进制文件没有反应，没有日志
> > > 系统：unbuntu 16.04/18.04
> > > 文件：ubuntu six-cli_V0.0.6-alpha_linux_amd64.tar.gz
> > > 直接编译报错
> > > ```
> > > go/src/github.com/Mrs4s/power-liner/shell.go:144:32: undefined: strings.ReplaceAll
> > > # github.com/Mrs4s/go-http-downloader
> > > go/src/github.com/Mrs4s/go-http-downloader/downloader.go:35:20: unknown field 'MaxConnsPerHost' in struct literal of type http.Transport
> > > # github.com/Mrs4s/go-six-cloud-upload-sdk
> > > go/src/github.com/Mrs4s/go-six-cloud-upload-sdk/uploader.go:32:20: unknown field 'MaxConnsPerHost' in struct literal of type http.Transport```
> > > 
> > > go版本为`go1.10 linux/amd64`
> > > ```
> > 
> > 
> > 运行无反应的问题还在排查
> > 编译要求最低的go版本是1.13, 请升级一下
> 
> 更换版本后编译生成的二进制文件可以正常使用，但是运行并退出后会影响ssh命令行中正常命令的显示，键入命令不会显示字符，这个问题在unbuntu16.04和ubuntu 18.04上都出现了。

目前这个问题是一个第三方库的bug，我暂时没时间解决。可以不使用shell模式，程序是支持six-cli.exe download /xxx/xxx.file 这种调用方式的。

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x28 pc=0x5c039b8e1c]

goroutine 1 [running]:
github.com/Mrs4s/six-cli/six_cloud.(*SixUser).GetFilesByPath(0x0, 0x5c039d30cc, 0x1, 0x4000192000, 0x4000040a00, 0x0, 0x1, 0x400018e0a0)
        /data/data/com.termux/files/home/storage/shared/6pan/six-cli/six_cloud/user.go:35 +0x144
github.com/Mrs4s/six-cli/shell/commands.filterCurrentDirs(0x4000190023, 0x1, 0x5c039d30cc)
        /data/data/com.termux/files/home/storage/shared/6pan/six-cli/shell/commands/handler.go:101 +0x40
github.com/Mrs4s/six-cli/shell/commands.PathCompleter(0x40001a6000, 0x4000192000, 0x1, 0x40001920f0, 0x7)
        /data/data/com.termux/files/home/storage/shared/6pan/six-cli/shell/commands/handler.go:64 +0x1a8
github.com/Mrs4s/six-cli/shell/commands.CommandHandler.JoinPathCompleter(...)
        /data/data/com.termux/files/home/storage/shared/6pan/six-cli/shell/commands/cd.go:59
reflect.callMethod(0x4000022840, 0x400015f148, 0x400015f120)
        /data/data/com.termux/files/usr/lib/go/src/reflect/value.go:717 +0x18c
reflect.methodValueCall(0x40001a6000, 0x4, 0x40001a6000, 0x4000064d80, 0x40001a4040, 0x2, 0x2, 0x0, 0x4000080000, 0x7b65a3c7d0, ...)
        /data/data/com.termux/files/usr/lib/go/src/reflect/asm_arm64.s:35 +0x28
github.com/Mrs4s/power-liner.(*Shell).RunAsShell.func1(0x4000190020, 0x4, 0x4, 0x20, 0x4000190020)
        /data/data/com.termux/files/home/go/pkg/mod/github.com/!mrs4s/power-liner@v0.0.0-20190813010520-d14dc4ea3290/shell.go:128 +0x3b0
github.com/peterh/liner.(*State).SetCompleter.func1(0x4000190014, 0x4, 0x4, 0x20, 0x4000190014, 0x4, 0x2, 0x5c03734178, 0x5, 0x4000000001)
        /data/data/com.termux/files/home/go/pkg/mod/github.com/peterh/liner@v1.1.0/common.go:198 +0x84
github.com/peterh/liner.(*State).tabComplete(0x4000128500, 0x400015f9f8, 0x11, 0x20, 0x400015f978, 0x4, 0x20, 0x4, 0x0, 0x0, ...)
        /data/data/com.termux/files/home/go/pkg/mod/github.com/peterh/liner@v1.1.0/line.go:367 +0x70
github.com/peterh/liner.(*State).PromptWithSuggestion(0x4000128500, 0x5c039d7610, 0x11, 0x0, 0x0, 0x4, 0x0, 0x0, 0x0, 0x0)
        /data/data/com.termux/files/home/go/pkg/mod/github.com/peterh/liner@v1.1.0/line.go:859 +0x310
github.com/peterh/liner.(*State).Prompt(...)
        /data/data/com.termux/files/home/go/pkg/mod/github.com/peterh/liner@v1.1.0/line.go:594
github.com/Mrs4s/power-liner.(*Shell).RunAsShell(0x400006e1e0)
        /data/data/com.termux/files/home/go/pkg/mod/github.com/!mrs4s/power-liner@v0.0.0-20190813010520-d14dc4ea3290/shell.go:170 +0x104
github.com/Mrs4s/six-cli/shell.RunAsShell(0x5c03b16440, 0x5c03da2040)
        /data/data/com.termux/files/home/storage/shared/6pan/six-cli/shell/shell.go:63 +0x578
main.main()
        /data/data/com.termux/files/home/storage/shared/6pan/six-cli/six.go:34 +0xd4
exit status 2
$
@Mrs4s 无法运行，Android10，Termux0.95，golang1.14.6/arm64

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -22,6 +22,7 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           GOARCH: amd64
+          CGO_ENABLED: 0 
           GOOS: linux
   release-darwin-386:
     name: release darwin/386

